frontend/src/theme/components/DatePicker.scss

```diff
@@ -0,0 +1,28 @@
+@import "node_modules/react-datepicker/src/stylesheets/datepicker.scss";
```

Comment:
woah, is this file necessary to use the DatePicker?

Reply:
It looks really strange without it: 
![image](https://user-images.githubusercontent.com/32009013/147863165-5fc3594e-0d96-400e-b191-d1516d7d77d9.png)

Reply:
wowsa, that is funky. How did you figure that this file was needed?